### ISSUE TITLE
sending a mail when a game is over.

### ISSUE THREAD

**Issue by o-jill** [o-jill/washcrus#63]
#62 was already implemented but you cannot know if the game is over.

when a game finished, you will send mail to both players, which are attached KIFU or have URL to download.

**Comment by o-jill** [comment]
attaching a file with mail gem.

```ruby
Mail.deliver do
  from     'me@test.lindsaar.net'
  to       'you@test.lindsaar.net'
  subject  'Here is the image you wanted'
  body     File.read('body.txt')
  add_file '/full/path/to/somefile.png'
end
```

or

```ruby
mail = Mail.new do
  from     'me@test.lindsaar.net'
  to       'you@test.lindsaar.net'
  subject  'Here is the image you wanted'
  body     File.read('body.txt')
  add_file :filename => 'somefile.png', :content => File.read('/somefile.png')
end

mail.deliver!
```

**Comment by o-jill** [comment]
implemented in 95c4ac8.
